lang/cs/Org.Apache.REEF.IMRU/OnREEF/Driver/IMRUDriver.cs

```diff
@@ -136,6 +137,7 @@ internal sealed class IMRUDriver<TMapInput, TMapOutput, TResult, TPartitionType>
         public void OnNext(IDriverStarted value)
         {
             RequestUpdateEvaluator();
+            RequestMapEvaluators(_totalMappers);
```

Comment:
How do we distinguish the `AllocatedEvaluator`s we get back? Absent a solution to that, we maybe should make IMRU assume that all Evaluators have the same resource spec?